webpack/move_to_foreman/common/helpers.js

```diff
@@ -14,7 +14,7 @@ const stringIsInteger = (value) => {
 
 export const getResponseErrorMsgs = ({ data }) => {
   if (data) {
-    const messages = (data.errors || data.displayMessage || data.message || data.error);
+    const messages = (data.errors || data.displayMessage || data.message || data.error.message);
```

Comment:
@beav I think this is a fair-enough approach until we have something like https://github.com/theforeman/foreman/pull/5814 merged so we can actually disable the UI components properly.

Do you know if we `data.error` is valid or perhaps it was added as a catch-all?

Reply:
Now it will show 'Access denied' instead of '[Object object]'

Reply:
I don't know the origins of `data.error` but if `data.error.message` works, I'd just go with that.